### ISSUE TITLE
Codeigniter 4 version

### ISSUE THREAD

**Issue by mrelliot69** [avenirer/CodeIgniter-MY_Model#301]
Do you have a version for codeigniter 4?

**Comment by svennd** [comment]
I don't think the author is planning a ci4 version, but in ci4 there is already a very basic model : https://codeigniter4.github.io/userguide/models/model.html
It's possible to extend from there. I kinda miss this in ci4 too !

**Comment by mrelliot69** [comment]
Yeah, i know we can extend .. i started by doing mymodel there .. but then found this one and tought would be great if we had this in codeigniter 4

**Comment by avenirer** [comment]
Nope. No plans on extending... :)
